client/src/components/board.rs

```diff
@@ -19,6 +12,14 @@ use dioxus::prelude::*;
 use futures::executor::block_on;
 use once_cell::sync::Lazy;
 
+use crate::arrows::{ArrowData, Arrows};
```

Comment:
Did rustrover like the `use crates` later?

Reply:
no, but i felt like having the crate stuff below other dependencies were better, just visually grouping our source code closer to the uses of our source code